src/parser/sql_keywords.txt

```diff
@@ -76,10 +76,11 @@ VALUES
 DIRECT
 SORTED
 
+COPY
```

Comment:
Does this file serve a purpose?

Reply:
I don't know, I just thought it is somehow common to update it